### ISSUE TITLE
Error about 'Could not load gatsby-config'

### ISSUE THREAD

**Issue by armdong** [gatsbyjs/gatsby-starter-hello-world#2]
When I run the `npm run develop`  command with my terminal, it occurs the 'Could not load gatsby-config' error, how can i fix it?

**Comment by armdong** [comment]
it works when I add the gatsby-config.js
``` javascript
module.exports = {
  siteMetadata: {
    title: `Gatsby starter hello world`,
  },
  plugins: [`gatsby-plugin-react-helmet`],
};
```

**Comment by RJFerguson** [comment]
I also encountered this issue, adding the gatsby-config.js as well as running 
`npm install --save gatsby-plugin-react-helmet`
fixed my issues.

**Comment by KyleAMathews** [comment]
This is a bug, sorry about this. `gatsby-config.js` shouldn't be required to run things :-( broke this earlier. Publishing a fix as we speak. Thanks for reporting!

**Comment by vincentmvdm** [comment]
Thank you so much @KyleAMathews! Was about to report this

**Comment by KyleAMathews** [comment]
Ok, fixed now in 1.8.11!
